app/Models/DocuSignToken.php

```diff
@@ -0,0 +1,144 @@
+<?php
+
+declare(strict_types=1);
+
+// phpcs:disable SlevomatCodingStandard.Functions.DisallowNamedArguments.DisallowedNamedArgument
+// phpcs:disable SlevomatCodingStandard.PHP.RequireExplicitAssertion.RequiredExplicitAssertion
+// phpcs:disable Generic.Commenting.DocComment.MissingShort
+
+namespace App\Models;
+
+use App\Exceptions\DocuSignTokenUnavailable;
+use App\Util\DocuSign;
+use Carbon\Carbon;
+use DocuSign\eSign\Client\Auth\OAuthToken;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Support\Facades\Cache;
+
+/**
+ * DocuSign tokens for the service account used to send envelopes.
+ *
+ * @property string $type
+ * @property string $token
+ * @property \Illuminate\Support\Carbon|null $created_at
+ * @property \Illuminate\Support\Carbon|null $updated_at
+ * @property \Illuminate\Support\Carbon $expires_at
+ *
+ * @method static \Illuminate\Database\Eloquent\Builder|DocuSignToken newModelQuery()
+ * @method static \Illuminate\Database\Eloquent\Builder|DocuSignToken newQuery()
+ * @method static \Illuminate\Database\Eloquent\Builder|DocuSignToken query()
+ * @method static \Illuminate\Database\Eloquent\Builder|DocuSignToken whereCreatedAt($value)
+ * @method static \Illuminate\Database\Eloquent\Builder|DocuSignToken whereExpiresAt($value)
+ * @method static \Illuminate\Database\Eloquent\Builder|DocuSignToken whereToken($value)
+ * @method static \Illuminate\Database\Eloquent\Builder|DocuSignToken whereType($value)
+ * @method static \Illuminate\Database\Eloquent\Builder|DocuSignToken whereUpdatedAt($value)
+ *
+ * @mixin \Barryvdh\LaravelIdeHelper\Eloquent
+ */
+class DocuSignToken extends Model
```

Comment:
This will get ripped out in favor of [JWT auth](https://developers.docusign.com/platform/auth/jwt/).